src/components/ContentHighlights/HighlightStepper/ContentHighlightStepper.jsx

```diff
@@ -100,11 +106,14 @@ const ContentHighlightStepper = ({ enterpriseId }) => {
         const trackInfo = {
           is_published: transformedHighlightSet.isPublished,
           highlight_set_uuid: transformedHighlightSet.uuid,
-          highlighted_content_uuids: transformedHighlightSet.highlightedContentUuids,
+          highlighted_content_uuids: transformedHighlightSet.highlightedContentUuids.map(highlight => ({
+            uuid: highlight.uuid,
+            aggregationKey: highlight.aggregationKey,
+          })),
         };
         sendEnterpriseTrackEvent(
           enterpriseId,
-          `${EVENT_NAMES.CONTENT_HIGHLIGHTS.STEPPER_STEP_CONFIRM_CONTENT}`,
+          `${EVENT_NAMES.CONTENT_HIGHLIGHTS.STEPPER_STEP_CONFIRM_CONTENT}.publish_button.clicked`,
```

Comment:
nit/curious: should these `.publish_button.clicked` and `.back.clicked` parts of the events names be part of the `EVENT_NAMES` constants (and elsewhere)?